camunda/variables/properties.py

```diff
@@ -0,0 +1,23 @@
+class Properties:
+    def __init__(self, properties={}):
+        self.properties = properties
+
+    def get_property(self, property_name):
+        prop = self.properties.get(property_name, None)
+        if not prop:
+            return None
+
+        return prop
```

Comment:
```suggestion
        return self.properties.get(property_name, None)
```
this should be sufficient. not need to check None